src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,55 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event that is happening on a certain date.
+ */
+public class Event extends Task implements Timeable {
+    protected LocalDate at;
```

Comment:
Might want to change this to private instead of protected. For java, unlike C++ where protected makes the variable inaccessible to other classes besides the derived classes, in java, as long as a class is in the same package, they can access the protected variable.

If that is not your intention, you may want to change to private since Event class is only using this variable (correct me if I'm wrong).

Reply:
Thank you for catching that error. I may have gotten it mixed up with experience in previous languages (like C#). 

I have resolved the other issues similar as the one mentioned